dbx/api/adjuster/policy.py

```diff
@@ -59,6 +63,25 @@ def _get_policy(policy_service: PolicyService, policy_name: Optional[str], polic
             f"Available policy names are: {[p.name for p in all_policies.policies]}"
         )
 
+    @staticmethod
+    def _should_merge_init_script() -> bool:
+        # currently only check on env var, in the future, we could also check on cli param
+        return getenv(constants.ENV_VAR_DBX_MERGE_INIT_SCRIPTS, "").lower() == "true"
```

Comment:
Please use the property setting via the `.dbx/project.json`, in the same way as `enable_inplace_jinja_support` is introduced.

Reply:
fixed